tools/Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM        ocaml/opam2:debian-10-ocaml-4.08
+FROM ocaml/opam2:debian-10-ocaml-4.08
 MAINTAINER  Christian Lindig <christian.lindig@citrix.com>
 
 COPY . /tmp/xs-opam
 
 RUN sudo apt-get update
 RUN opam repo remove --all default \
- && opam repo add xs-opam file:///tmp/xs-opam \
+ && opam repo add xs-opam --all-switches file:///tmp/xs-opam \
+ && opam switch 4.08 \
```

Comment:
4.08.1?

Reply:
When I tested it locally, it worked. Not sure how significant this is.

Reply:
ignore this then :+1: